lib/github-pages-health-check/domain.rb

```diff
@@ -410,6 +410,8 @@ def cname
         cnames = dns.take_while { |answer| answer.type == Dnsruby::Types::CNAME }
         return if cnames.empty?
 
+        # check to see if the CNAME starts with www
+        @wwwcname ||= cnames.last.name.to_s.start_with?("www")
```

Comment:
I think we might want this to be `www.` so we don't match on `wwwblog.cname.com`?
```suggestion
        @wwwcname ||= cnames.last.name.to_s.start_with?("www.")
```